scripts/update_schemastore.py

```diff
@@ -12,8 +12,8 @@
 from subprocess import check_call, check_output
 from tempfile import TemporaryDirectory
 
-schemastore_fork = "git@github.com:astral-sh/schemastore.git"
-schemastore_upstream = "git@github.com:SchemaStore/schemastore.git"
+schemastore_fork = "https://github.com/astral-sh/schemastore.git"
+schemastore_upstream = "https://github.com/SchemaStore/schemastore.git"
```

Comment:
You can't just change it, it'll break for ssh authentication users i.e. Charlie haha

Reply:
Oof yeah, right

Reply:
I had the same idea initially :D https://github.com/astral-sh/ruff/pull/5322

Reply:
A simple `--proto ssh|https` flag seems sufficient?

Reply:
Added a new CLI flag in https://github.com/astral-sh/ruff/pull/11353/commits/6e7b3dbc07e33f8a32b6126c2ff06e6827885db8, how's that look?